cfme/containers/image.py

```diff
@@ -140,9 +139,8 @@ def all(self):
             ems_table, image_table.ems_id == ems_table.id)
         images = []
 
-        for name, id, provider_name in image_query.all():
-            images.append(self.instantiate(name=name, id=id,
-                provider=ContainersProvider(name=provider_name, appliance=self.appliance)))
+        for name, id in image_query.all():
+            images.append(self.instantiate(name=name, id=id, provider=provider))
```

Comment:
@gshefer Note here: I changed this from instantiating a generic containers provider to the provider under test